mlflow/genai/scorers/base.py

```diff
@@ -122,9 +123,12 @@ def filter_string(self) -> str | None:
     def status(self) -> ScorerStatus:
         """Get the status of this scorer, using only the local state."""
 
-        if self.sample_rate is None:
+        if self._registered_backend is None:
             return ScorerStatus.UNREGISTERED
 
+        if self._registered_backend == "tracking":
+            return ScorerStatus.STOPPED
+
         return ScorerStatus.STARTED if self.sample_rate > 0 else ScorerStatus.STOPPED
```

Comment:
```suggestion
        if self._registered_backend is None:
            return ScorerStatus.UNREGISTERED

        return ScorerStatus.STARTED if (self.sample_rate or 0) > 0 else ScorerStatus.STOPPED
```

@WeichenXu123 Would this approach work? This way, we don't have to distinguish between Databricks and non-Databricks backend.